config.toml

```diff
@@ -8,27 +8,37 @@ relativeURLs = false
 
 [permalinks]
     post = "/:year/:month/:day/:slug/"
-
 [[menu.main]]
-    identifier = "main"
-    name = "Delphi"
+    identifier = "about"
+    name = "About"
     url = "https://delphi.cmu.edu/"
```

Comment:
please use a relative link here `/about` isn't it?

Reply:
Hmm I'm not sure if matching the active url to these urls to highlight active tab will work if these are changed to be relative, might have to rethink that then if it's important to make these relative.

Reply:
Actually I take that back, it looks like it works.